### ISSUE TITLE
Traefik 1.3.6 is setting the X-Forwarded-Port to the private port instead of the public port

### ISSUE THREAD

**Issue by dduportal** [containous/traefik#2000]
### Do you want to request a *feature* or report a *bug*?

Reporting a *bug*

### What did you do?

Traefik updated from 1.3.5 to 1.3.6 (docker image).

My backend is now receiving the header `X-Forwarded-Port`.
But the value is not the port requested by the public client, 
but it the private port of the backend.

```
Client -> Traefik (443) -> Backend (8080)
```

### What did you expect to see?

 `X-Forwarded-Port` should be equals to 443

### What did you see instead?

 `X-Forwarded-Port` equals to 8080

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      v1.3.6
Codename:     raclette
Go version:   go1.8.3
Built:        2017-08-22_09:57:54AM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```json
{
 "GraceTimeOut": 10000000000,
 "Debug": false,
 "CheckNewVersion": true,
 "AccessLogsFile": "",
 "TraefikLogsFile": "",
 "LogLevel": "ERROR",
 "EntryPoints": {
  "http": {
   "Network": "",
   "Address": ":80",
   "TLS": null,
   "Redirect": {
    "EntryPoint": "https",
    "Regex": "",
    "Replacement": ""
   },
   "Auth": null,
   "Compress": false
  },
  "https": {
   "Network": "",
   "Address": ":443",
   "TLS": {
    "MinVersion": "",
    "CipherSuites": null,
    "Certificates": null,
    "ClientCAFiles": null
   },
   "Redirect": null,
   "Auth": null,
   "Compress": false
  }
 },
 "Cluster": null,
 "Constraints": [],
 "ACME": {
  "Email": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
  "Domains": [
   {
    "Main": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
    "SANs": null
   },
   {
    "Main": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
    "SANs": null
   }
  ],
  "Storage": "",
  "StorageFile": "/acme/acme.json",
  "OnDemand": true,
  "OnHostRule": true,
  "CAServer": "",
  "EntryPoint": "https",
  "DNSProvider": "",
  "DelayDontCheckDNS": 0,
  "ACMELogging": false,
  "TLSConfig": null
 },
 "DefaultEntryPoints": [
  "https",
  "http"
 ],
 "ProvidersThrottleDuration": 2000000000,
 "MaxIdleConnsPerHost": 200,
 "IdleTimeout": 180000000000,
 "InsecureSkipVerify": false,
 "Retry": null,
 "HealthCheck": {
  "Interval": 30000000000
 },
 "Docker": null,
 "File": null,
 "Web": null,
 "Marathon": null,
 "Consul": null,
 "ConsulCatalog": null,
 "Etcd": null,
 "Zookeeper": null,
 "Boltdb": null,
 "Kubernetes": null,
 "Mesos": null,
 "Eureka": null,
 "ECS": null,
 "Rancher": null,
 "DynamoDB": null,
 "ConfigFile": "/etc/traefik/traefik.toml"
}
```

### If applicable, please paste the log output in debug mode (`--debug` switch)

Not applicable

**Comment by traefiker** [comment]
Closed by #2007.

**Comment by dduportal** [comment]
👍
